staging/src/k8s.io/kubectl/pkg/cmd/top/top.go

```diff
@@ -39,9 +39,27 @@ var (
 	topLong = templates.LongDesc(i18n.T(`
 		Display resource (CPU/memory) usage.
 
-		The top command allows you to see the resource consumption for nodes or pods.
+		This command provides a real-time view of resource consumption for nodes and pods.
```

Comment:
Not sure about "real-time", might confuse user that values don't change, as metrics are collected on period of 15s.